JitsiConference.js

```diff
@@ -332,6 +332,16 @@ JitsiConference.prototype.addTrack = function (track) {
     this.room.addListener(XMPPEvents.SENDRECV_STREAMS_CHANGED,
         track.ssrcHandler);
 
+    // Report active device to statistics
+    var devices = RTC.getCurrentlyAvailableMediaDevices();
+    device = devices.find(function (d) {
```

Comment:
There is already similar code in JitsiLocalTrack._setRealDeviceIdFromDeviceList method: https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/RTC/JitsiLocalTrack.js#L107. We can probably expose it as public method and re-use